tests/foreman/ui/test_rhcloud_insights.py

```diff
@@ -304,10 +304,7 @@ def test_host_sorting_based_on_recommendation_count():
 
 
 @pytest.mark.no_containers
-@pytest.mark.rhel_ver_list(r'^[\d]+$')
-@pytest.mark.parametrize(
-    "module_target_sat_insights", [True, False], ids=["hosted", "local"], indirect=True
-)
+@pytest.mark.rhel_ver_list([10])
```

Comment:
Would this work too? (if you need just the latest & greatest)
```suggestion
@pytest.mark.rhel_ver_match('N-0')
```